### ISSUE TITLE
Keypresses are registering while typing in text inputs

### ISSUE THREAD

**Issue by mickdekkers** [volundr-/imabotlol#3]
This makes it impossible to type a name containing the letters `a`, `d`, `t`, or `c`.

**Comment by Cheeseskates** [comment]
can't you just untoggle them when your done typing your name? :S

**Comment by mickdekkers** [comment]
No, because if you press `a` when typing the name for example, it exits out of the menu and starts the bot with the name "imabotlol"
